modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-query_{context}"]
+= Query configuration options
+
+Query is a service that retrieves traces from storage and hosts the user interface to display them.
+
+.Parameters used by the {TempoOperator} to define Query
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|spec:
+  query:
+    replicas:
+|Specifies the number of Query replicas to create.
+|Positive integer
+|
+|===
+
+
+.Configuration parameters passed to Query
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|spec:
+  query:
+    options: {}
+|Configuration options that define the Query service.
+|
+|
+
+|options:
+  log-level:
+|Logging level for Query.
+|`debug`, `info`, `warn`, `error`, `fatal`, `panic`
+|
+
+|options:
+  query:
+    base-path:
+|You can set the base path for all tempo-query HTTP routes to a non-root value: for example, `/tempo` will cause all UI URLs to start with `/tempo`. This can be useful when running `tempo-query` behind a reverse proxy.
+|/<path>
+|
+|===
+
+.Sample Query configuration
+====
```

Comment:
This section renders in a box. (====). Removing it [54](https://github.com/openshift/openshift-docs/pull/62410/files#diff-8d5d8278cc5ae4e12161d84347dbb1d8473d74919257fa84ea2aa0156b125d46R54) and [69](https://github.com/openshift/openshift-docs/pull/62410/files#diff-8d5d8278cc5ae4e12161d84347dbb1d8473d74919257fa84ea2aa0156b125d46R69) should fix the rendering
Related: https://github.com/openshift/openshift-docs/pull/62410#discussion_r1318992887

Reply:
Thank you. Already fixed in response to another comment.